db/models.py

```diff
@@ -98,6 +99,9 @@ def __repr__(self):
         values = (self.app_id, self.name)
         return str_format % values
         
+    def toJSON(self):
```

Comment:
@algorithmike If I remember correctly, lines 102-105 do not work. They should be removed.